src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ExtensionDataReader.cs

```diff
@@ -82,13 +83,17 @@ internal void Reset()
             _elements = null;
         }
 
+#pragma warning disable CS8775 // Member must have a non-null value when exiting in some condition.
```

Comment:
you can also add Debug.Assert to remove the warning

Reply:
Is there a preferred way? For this case, it feels like I'd have to change a lot of code to use a Debug.Assert.